src/client/game.cpp

```diff
@@ -1238,8 +1238,6 @@ void Game::shutdown()
 	if (g_touchscreengui)
 		g_touchscreengui->hide();
 
-	showOverlayMessage(N_("Shutting down..."), 0, 0, false);
```

Comment:
Removing this results in a delay before the shutdown screen is shown while `client` etc. are being destructed.

Reply:
However, having this results in a black screen that is only displayed for the fraction of a second on my computer…
Anyway, I think my most recent commit should've fixed that